### ISSUE TITLE
Resolve Infraspecies spreadsheet notes before import.

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Herbarium-Sheets-workstation#114]
The infraspecies spreadsheet, where we recorded any missing taxonomic information for Box 1-285, has been filled in (#91). However, some information couldn't be added into the taxon fields and was therefore written in the notes field. These notes need to be resolved before we can import it into Specify.

These issues are:

- s.l. / cf. There is an issue for this in general, https://github.com/orgs/NHMDenmark/projects/29?pane=issue&itemId=61683381

- Hybrid names ("Dahlia x cultorum Thorsr. & Reis.", "Doronicum x excelsum (N.E.Br.) Stace"), so species where they gave a new name for a hybrid. Can this go into Specify as it is?

- "Doronicum sp. Orientale/plantagineum? Evt. D. p. var. excelsum N. E. Brown? (Se Karlsson 1885)". How should we treat this? What exactly does it even mean, is everything below Genus level uncertain?

We need to discuss these with the Collection Managers and decide what exactly should be added to Specify and how we do this. Can we leave the things in the notes field for now, and import it into Specify with the other information and then clean it up once we have a permanent solution for these cases?

**Comment by PipBrewer** [comment]
@RebekkaML Yes we can import and deal with later. We need to make sure that we have a mechanism for following up on these.

**Comment by RebekkaML** [comment]
> @RebekkaML Yes we can import and deal with later. We need to make sure that we have a mechanism for following up on these. If Jens is on Github, we could tag him.

Ok, so for now we can keep these things in the notes column in the infraspecies spreadsheet. I would then make a new issue (or see if one already exists) for making a general protocol of how to deal with these cases once they are imported into Specify.

**Comment by RebekkaML** [comment]
I created the Issue https://github.com/orgs/NHMDenmark/projects/29/views/1?pane=issue&itemId=74923800 for resolving this kind of comment in Specify and the infraspecies spreadsheet will be imported with the comments.
